frontend/public/style/ancillary/_bootstrap-variables.scss

```diff
@@ -4,54 +4,51 @@
 
 //== Typography
 
-$font-size-base:          14px !default;
-$font-size-large:         ceil(($font-size-base * 1.25)) !default; // ~18px
-$font-size-small:         ceil(($font-size-base * .85)) !default; // ~12px
+$font-size-base: 14px !default;
+$font-size-large: ceil(($font-size-base * 1.25)) !default; // ~18px
+$font-size-small: ceil(($font-size-base * 0.85)) !default; // ~12px
 
 //** Unit-less `line-height` for use in components like buttons.
-$line-height-base:        1.428571429 !default; // 20/14
+$line-height-base: 1.428571429 !default; // 20/14
 //** Computed "line-height" (`font-size` * `line-height`) for use with `margin`, `padding`, etc.
-$line-height-computed:    floor(($font-size-base * $line-height-base)) !default; // ~20px
+$line-height-computed: floor(($font-size-base * $line-height-base)) !default; // ~20px
 
 //== Components
 
-$line-height-large:         1.3333333 !default; // extra decimals for Win 8.1 Chrome
-$line-height-small:         1.5 !default;
+$line-height-large: 1.3333333 !default; // extra decimals for Win 8.1 Chrome
+$line-height-small: 1.5 !default;
 
 //== Forms
 
-//** `.form-group` margin
-$form-group-margin-bottom:       15px !default;
-
 //-- Z-index master list
-$zindex-navbar:            1000 !default;
-$zindex-navbar-fixed:      1030 !default;
-$zindex-modal:             1050 !default;
+$zindex-navbar: 1000 !default;
+$zindex-navbar-fixed: 1030 !default;
+$zindex-modal: 1050 !default;
 
 //== Media queries breakpoints
 
 // Extra small screen / phone
 //** Deprecated `$screen-xs` as of v3.0.1
-$screen-xs:                  480px !default;
```

Comment:
wonder if we can use `pf-v6-global--breakpoint` sass variables instead here

Reply:
We want to maintain these for the time being as there are quite a layouts that would need to be tested since these values differ from the PatternFly ones.